secureserver.net.professional_mail.json

```diff
@@ -53,11 +53,32 @@
     },
     {
       "groupId": "verification",
-      "groupId": "txt",
       "type": "TXT",
       "host": "@",
       "data": "%txtValue%",
       "ttl": 3600
+    },
+    {
+      "groupId": "dmarc",
+      "type": "TXT",
+      "txtConflictMatchingPrefix": "v=DMARC1;",
```

Comment:
This one without txtConflictMatchingMode is void actually. @kerolasa maybe Linter could catch such issue and warn.
On other note for DMARC being a distinct host likely All setting instead of Prefix will work as well?

Reply:
fixed. thank you